comp/logs/agent/flare/flare_controller.go

```diff
@@ -38,7 +38,7 @@ func (fc *FlareController) FillFlare(fb flaretypes.FlareBuilder) error {
 	if len(fc.allFiles) == 0 && len(fc.journalFiles) == 0 {
 		return nil
 	}
-	fb.AddFileFromFunc("logs_file_permissions.log", func() ([]byte, error) {
+	fb.AddFileFromFunc("logs_file_permissions.log", func() ([]byte, error) { //nolint:errcheck
```

Comment:
Why the `nolint` became necessary? I don't see any other changes in the code.

Reply:
I'm not sure, but it appears that the linter changed at some point, and complained about these usages when I put this PR into the CI.

Reply:
Looking at the rest of the PR, looks like you had to add a lot more than just here. This doesn't look right. Would it make sense to raise the issue about the linter change? A feature PR shouldn't have to fix lints in a lot of unrelated places.